ui/lib/pki/addon/components/page/pki-key-list.hbs

```diff
@@ -0,0 +1,61 @@
+<Toolbar>
+  <ToolbarActions>
+    {{#if @canImportKey}}
+      <ToolbarLink @route="keys.import" @type="download">
+        Import
+      </ToolbarLink>
+    {{/if}}
+    {{#if @canGenerateKey}}
+      <ToolbarLink @route="keys.create" @type="add">
+        Generate
+      </ToolbarLink>
+    {{/if}}
+  </ToolbarActions>
+</Toolbar>
+<p class="has-padding">Below is information about the private keys used by the issuers to sign certificates. While
+  certificates represent a public assertion of an identity, private keys represent the private part of that identity, a
+  secret used to prove who they are and who they trust.</p>
+
+{{#if @keyModels.length}}
+  {{#each @keyModels as |pkiKey|}}
+    <LinkedBlock class="list-item-row" @params={{array "keys.key.details" pkiKey.keyId}} @linkPrefix={{@mountPoint}}>
+      <div class="level is-mobile">
+        <div class="level-left">
+          <div>
+            <Icon @name="certificate" class="has-text-grey-light" />
+            <span class="has-text-weight-semibold is-underline">
+              {{pkiKey.id}}
+            </span>
+            <div class="is-flex-row has-left-margin-l has-top-margin-xs">
+              {{#if pkiKey.keyName}}
+                <span class="tag has-text-grey-dark">{{pkiKey.keyName}}</span>
+              {{/if}}
+            </div>
+          </div>
+        </div>
+        <div class="level-right is-flex is-paddingless is-marginless">
+          <div class="level-item">
+            <PopupMenu>
+              <nav class="menu">
+                <ul class="menu-list">
+                  <li>
+                    <LinkTo @route="keys.key.details" @model={{pkiKey.keyId}} @disabled={{eq pkiKey.canRead false}}>
```

Comment:
Since the boolean is calculated on the model you could use `not` instead of `eq ... false`. Also do we want to used a passed arg here?
```suggestion
                    <LinkTo @route="keys.key.details" @model={{pkiKey.keyId}} @disabled={{not @canRead}}>
```

Reply:
yeah - I was wondering about doing something similar to what I did for `@canGenerateKey` so like the following. If we are okay with reading those capabilities just based on the first model?
```js
@canReadKey={{this.model.keyModels.firstObject.canReadKey}}

Reply:
I thought we didn't want to use `not` in case the capability is `undefined` for some reason?

Reply:
oohhh ^ disregard, forgot its explicitly checking for `false` in the model getter now